### PR TITLE
fix(progressbar): fix progressbar issue

### PR DESCRIPTION
**Pull request by pedrohmorais**
## Description
Fix a issue that breaks quantum when it is builded.

## Setup
To view this error, just install and import quantum in a application. Or try to use a builded version.

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests: ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [] Code review

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] IE 11
  - [x] Firefox
  - [x] Chrome
  - [x] Safari
  - [x] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/shared/hexToRgb.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const hexToRgb = hex => {
```
Comment (rapahaeru) [inline components/shared/hexToRgb.js]: Testing the implementation, i noticed that method is not used in any place.
What do you think to just remove it?

File: components/shared/index.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import BREAKPOINTS from './breakpoints';
 import uniqId from './uniqId';
 import * as INPUT_STYLE from './inputStyle';
 import hexToRgba from './hexToRgba';
+import hexToRgb from './hexToRgb';
```
Comment (rapahaeru) [inline components/shared/index.js]: we don't need this anymore, if i am correct.

Comment (renatogalvones) [reply components/shared/index.js]: you're correct! :wink:

File: components/shared/hexToRgb.js (hunk 1 above)
Comment (pedrohmorais) [reply components/shared/hexToRgb.js]: Good idea!

File: components/shared/index.js (hunk 2 above)
Comment (pedrohmorais) [reply components/shared/index.js]: Removed

**Pull request merged**